src/test/java/com/microsoft/graph/serializer/DefaultSerializerTest.java

```diff
@@ -33,4 +35,32 @@ public void testDeserializationOfObjectWithODataTypeProperty() {
         Mockito.verify(reaction.additionalDataManager()).setAdditionalData(gson.fromJson(testJsonResponse, JsonElement.class).getAsJsonObject());
     }
 
+    @Test
+    public void testDefaultSerializerDoesNotIncludeNullValuesByDefault() {
+        // Given
+        final String testJsonResponse =
+            "{\"@odata.type\": \"#microsoft.graph.messageStub\", \"body\": null}";
+
+        // When
+        DefaultSerializer nonNullSerializer = new DefaultSerializer(logger);
```

Comment:
please add final modifiers for both variables (both tests)

Reply:
also: maybe you can rely on the defaultSerializer field here since it's initialized the same way instead of instantiating a new one.